Localize/lang/strings.json

```
@@ -700,6 +697,7 @@
   "W070M2": "of {max}",
   "W6FdMh": "Required. The name of the new property.",
   "WGwH45": "Clear",
+  "WK6hZX": "Select all text in {label}",
   "WLj8DH": "Please enter a valid dictionary",
```

"Please" isn't necessary or appropriate:

```suggestion
  "WLj8DH": "Enter a valid dictionary.",
```